install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1772,7 +1772,7 @@ loadBalancer:
   # mode: snat
 
   # -- acceleration is the option to accelerate service handling via XDP
-  # e.g. native, disabled
+  # e.g. native, disabled, generic, best-effort
```

Comment:
As @mhofstetter, it would be nice to have a word or two explaining what these different options are (or, a link to the documentation). Helm chart docs, sadly, don't well with paragraph breaks / formatting because of some silly transformations we do :crying_cat_face:. So, if documenting this gets hard, it should be a section in the XDP documentation.

Sorry, I know this is a bit more work than you bargained for.

Reply:
No worries - I will take care of this either way

Reply:
Also updated the regular documentation (Documentation/bpf/progtypes.rst) in commit 305dfa694c82987ab69733e9b1e199f672b44350